walkthroughs/2-fuse-aggregator-and-api-management/walkthrough.adoc

```diff
@@ -49,11 +49,11 @@ image::images/arch.png[integration, role="integr8ly-img-responsive"]
 ****
 
 [type=walkthroughResource,serviceName=codeready]
-.CodeReady Workspaces
+.Code Ready
```

Comment:
I think the correct name is the original one as discussed in https://issues.jboss.org/browse/INTLY-1208

Reply:
@joaedwar yep, that jira gives the correct names as per https://access.redhat.com/documentation/en-us/red_hat_codeready_workspaces_for_openshift/1.0.0/